lxd/main_recover.go

```diff
@@ -229,7 +229,7 @@ func (c *cmdRecover) Run(cmd *cobra.Command, args []string) error {
 	fmt.Print("Starting recovery...\n")
 
 	// Send /internal/recover/import request to LXD.
-	reqImport := internalRecoverImportPost{
+	reqImport := internalRecoverImportPost{ //nolint:gosimple
```

Comment:
Why did this need nolinting?

Reply:
Oh this was because `reqValidate` is of type `internalRecoverValidatePost` which is currently identical to `internalRecoverImportPost` so gosimple was recommending a type conversion. I figured that was less clear than what we have currently, and wouldn't work if either type was changed so I just nolinted it.

Shall I add a comment to explain it?

Reply:
That would be good thanks, makes sense.